### PR TITLE
Add rp_mode to pytest.ini

### PR DESCRIPTION
**Pull request by kevin-henry-fernride**

### CODE REVIEW

File: pytest_reportportal/config.py [hunk 1]
```diff
@@ -30,12 +30,16 @@ def __init__(self, pytest_config):
         self.rp_launch_description = self.find_option('rp_launch_description')
         self.rp_log_batch_size = int(self.find_option('rp_log_batch_size'))
         self.rp_log_level = get_actual_log_level(self.pconfig, 'rp_log_level')
-        self.rp_parent_item_id = self.find_option('rp_parent_item_id')
-        self.rp_project = self.find_option('rp_project')
-        self.rp_rerun_of = self.find_option('rp_rerun_of')
-        self.rp_retries = int(self.find_option('retries'))
-        self.rp_uuid = getenv('RP_UUID') or self.find_option('rp_uuid')
-        self.rp_verify_ssl = self.find_option('rp_verify_ssl')
+        self.rp_parent_item_id = (self.pconfig.option.rp_parent_item_id or
```
Comment (iivanou) [inline pytest_reportportal/config.py:33]: Please, use the find_option method. Rebase against the master.

File: pytest_reportportal/service.py [hunk 2]
```diff
@@ -700,8 +700,9 @@ def _get_item_description(test_item):
...
-            doc = test_item.obj.__doc__
-            if doc is not None:
-                return trim_docstring(doc)
+            if hasattr(test_item, "obj"):
+                doc = test_item.obj.__doc__
+                if doc is not None:
+                    return trim_docstring(doc)
         if isinstance(test_item, DoctestItem):
-            return test_item.reportinfo()[2]
+            return test_item.reportinfo()[2]
```
Comment (iivanou) [inline pytest_reportportal/service.py:708]: Add a new line at the end of the file.

File: pytest_reportportal/service.py [hunk 3]
```diff
@@ -700,8 +700,9 @@ def _get_item_description(test_item):
         :return string description
         """
         if isinstance(test_item, (Class, Function, Module, Item)):
-            doc = test_item.obj.__doc__
-            if doc is not None:
-                return trim_docstring(doc)
+            if hasattr(test_item, "obj"):
```
Comment (iivanou) [inline pytest_reportportal/service.py:703]: Please, add a unit test for this method.

**Review by iivanou — CHANGES_REQUESTED** [review]

**Comment by HardNorth** [comment]
Abandoned

**Pull request closed without merging**